plugins/flutter_aepedgeidentity/android/src/main/java/com/adobe/marketing/mobile/flutter/flutter_aepedgeidentity/FlutterAEPEdgeIdentityPlugin.java

```diff
@@ -0,0 +1,179 @@
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+package com.adobe.marketing.mobile.flutter.flutter_aepedgeidentity;
+
+import android.annotation.SuppressLint;
+import android.util.Log;
+
+import com.adobe.marketing.mobile.AdobeError;
+import com.adobe.marketing.mobile.AdobeCallbackWithError;
+import com.adobe.marketing.mobile.edge.identity.Identity;
+import com.adobe.marketing.mobile.edge.identity.IdentityMap;
+import com.adobe.marketing.mobile.edge.identity.IdentityItem;
+import androidx.annotation.NonNull;
+import io.flutter.embedding.engine.plugins.FlutterPlugin;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.MethodChannel.MethodCallHandler;
+import io.flutter.plugin.common.MethodChannel.Result;
+
+import java.util.Map;
+import java.util.List;
+import java.util.ArrayList;
+
+/** FlutterAEPEdgeIdentityPlugin */
+public class FlutterAEPEdgeIdentityPlugin implements FlutterPlugin, MethodCallHandler {
+
+  private static final String TAG = "FlutterAEPEdgeIdentityPlugin";
+  private MethodChannel channel;
+  
+
+  @Override
+  public void onAttachedToEngine(@NonNull final FlutterPluginBinding binding) {
+    channel = new MethodChannel(binding.getBinaryMessenger(), "flutter_aepedgeidentity");
+    channel.setMethodCallHandler(new FlutterAEPEdgeIdentityPlugin());
+  }
+
+  @Override
+  public void onDetachedFromEngine(@NonNull final FlutterPluginBinding binding) {
+    if (channel != null) {
+      channel.setMethodCallHandler(null);
+    }
+  }
+
+  @Override
+  public void onMethodCall(MethodCall call, Result result) {
+    if ("extensionVersion".equals(call.method)) {
+      AndroidUtil.runOnUIThread(new Runnable() {
+        @Override
+        public void run() {
+           result.success(Identity.extensionVersion());
+        }
+      });
+    } else if ("getExperienceCloudId".equals(call.method)) {
+         handleGetExperienceCloudId(result);
+    } else if ("getUrlVariables".equals(call.method)) {
+         handlerGetUrlVariables(result);
+    } else if ("getIdentities".equals(call.method)) {
+          handlerGetIdentities(result);
+    } else if ("updateIdentities".equals(call.method)) {
+         handlerUpdateIdentities(call.arguments);
+         result.success(null);
+    } else if ("removeIdentities".equals(call.method)) {
+         Map item = call.argument("item");
+         String namespace = call.argument("namespace");
+         handlerRemoveIdentities(item, namespace);
+         result.success(null);
+  }
+    else {
+      AndroidUtil.runOnUIThread(new Runnable() {
+        @Override
+        public void run() {
+           result.notImplemented();
+        }
+      });
+    }
+  }
+
+  private void handleGetExperienceCloudId(final MethodChannel.Result result) {
```

Comment:
what is the standard `handle` or `handler` prefix?

Reply:
Use handle for all now.